.github/workflows/ci.yml

```diff
@@ -61,7 +61,6 @@ jobs:
         allow-failure:
           - false
         scenario:
-          - ember-lts-3.24
           - ember-lts-3.28
```

Comment:
We could have also just run `ember-cli-update` here, which would both drop 3.24 and node 12. Anyway, separate PRs is even better, but then we should run `ember-cli-update` nevertheless, especially to update the test matrix here. We should have the 4.x LTS releases covered here. All of which can happen in a separate PR of course...

Reply:
I prefer to have breaking changes in separate PRs as it makes it easier for consumers to review. I can run `ember-cli-update` later. :+1: